fed-budget-chart/js/lines.js

```diff
@@ -0,0 +1,265 @@
+        enabled: false,
+        align: 'right',
+        verticalAlign: 'middle',
+        width: '20%'
+      },
+      tooltip: {
+        outside: true,
+        useHTML: true,
+        shared: false,
+        valueDecimals: 3,
+        headerFormat: '<span style="font-size: 14px"><b>FY {point.key}</b></span><br/>',
+        pointFormatter: function () {
+          var result = this.y
+          if (result > 999999.99) {
+            result = (result / 1000000).toFixed(2) + " Trillion"
+          } else if (result > 999.99) {
+            result = (result / 1000).toFixed(2) + " Billion"
+          } else if (result > 0) {
+            result = result.toFixed(2) + " Million"
+          } else if (result < 0) {
+            result = Math.abs((result / 1000)).toFixed(2) + " Billion"
+            return '<span style="color:' + this.color + '">\u25CF</span> ' + this.series.name + ': <b>-$' + result + '</b><br/>'
+          }
+          return '<span style="color:' + this.color + '">\u25CF</span> ' + this.series.name + ': <b>$' + result + '</b><br/>'
+        },
+        style: {
+          whitespace: 'normal'
+        },
+      },
+      plotOptions: {
+        line: {
+          marker: {
+            enabled: false,
+            symbol: "circle",
+            radius: 3,
+          },
+          lineWidth: 1,
+        },
+      },
+      series: data.values,
+      },
+      function (chart) {
+        setUpButtons(chart)
+        setUpCheckboxes(chart)
+      });
+}
+
+function setUpDropdown(values) {
+  const select = document.getElementById("dataset");
+  let optionsHTML = "";
+
+  values.forEach((element) => {
+    optionsHTML += `<option value="${element.name}">${element.name}</option>`;
+  });
+
+  select.innerHTML = optionsHTML;
+
+  select.addEventListener("change", function () {
+    let chart = Highcharts.chart("hcContainer", {});
+    chart.destroy();
+    renderChart(allData[this.value]);
+  });
+}
+
+function setUpButtons(chart) {
+  let series = chart.series
+
+  const selectAll = document.getElementById("select-all");
+  const unselectAll = document.getElementById("unselect-all");
+
+  selectAll.addEventListener("click", function () {
+    for (let i = 0; i < series.length; i++) {
+      series[i].setVisible(true, true);
+      setUpCheckboxes(chart)
+    }
+  })
+
+  unselectAll.addEventListener("click", function () {
+    for (let i = 0; i < series.length; i++) {
+      series[i].setVisible(false, false)
+      setUpCheckboxes(chart)
+    }
+  })
+}
+
+function setUpCheckboxes(chart) {
+  const checkboxes = document.getElementById("checkboxes")
+  let series = chart.series
+  let checkboxesHTML = ""
+
+  for (let i = 0; i < series.length; i++) {
+    let isChecked = series[i].visible ? 'checked' : ""
+    checkboxesHTML += `
+    <div class="checkbox__wrapper" style="--color: ${series[i].color}">
+      <input type="checkbox" name="series" id="${i}" value="${i}" ${isChecked} style="--color: ${series[i].color}" />

```

Comment:
You shouldn't need to add the inline style here. It will inherit from it's parent (the `.checkbox__wrapper`).

Reply:
🤔 I took it out of <input> but if I take it out of the <div> all of them turn black